src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const listEmployees = document.querySelectorAll('[data-salary]');
+
+const sortList = (list) => {
+  const UL = document.querySelector('ul');
+
+  const orderedListEmployees = [...list].sort((a, b) => {
+    const salaryA = a.getAttribute('data-salary').replace(/\D/g, '');
```

Comment:
```suggestion
    const salaryA = a.getAttribute('data-salary').replace(/\D/g, '');
```
створи функцію parseSalary